cf/query.py

```diff
@@ -645,7 +711,7 @@ def value(self):
 
     def addattr(self, attr):
         """Return a `Query` object with a new left hand side operand
-        attribute to be used during evaluation. TODO.
+        attribute to be used during evaluation.
```

Comment:
I think that this TODO was to probably to make this fit on one line (as PEP8 suggests). How about:
```
        """Redefine the query to be on an object's attribute.
```

Reply:
I see. I just thought it would show up in the docs and seem ugly so I blitzed it.

Great, that is always good practice. Nice suggestion, I will include it in a new commit to push up.

Reply:
Done in 030e5cf5df48a5bca0cb240d139206f405fc329c. Marking as resolved.